packages/angular_devkit/build_angular/src/tools/esbuild/angular/compiler-plugin.ts

```diff
@@ -420,7 +421,16 @@ export function createCompilerPlugin(
         for (const { outputFiles, metafile } of additionalResults.values()) {
           // Add any additional output files to the main output files
           if (outputFiles?.length) {
-            result.outputFiles?.push(...outputFiles);
+            const outdir = build.initialOptions.outdir;
+            result.outputFiles?.push(
+              ...outputFiles.map((output) => {
+                if (outdir) {
+                  //  output.path = path.join(outdir, output.path);
+                }
+
+                return output;
+              }),
+            );
```

Comment:
This block does not appear to be doing anything.

Reply:
Yeah this is a left over. I’ll remove it